src/styles/base.scss

```
@@ -0,0 +1,48 @@
+@import url('fonts.css');
+
+$base-font:         "Ubuntu", sans-serif;
+$base-font-size:    14px;
+$base-color:        #1E1E1E;
+$accent-color:      #F9F144;
+$mark-color:        #E4EDFD;
+$ext-color:         #FBFBFB;
+$note-color:        #999999;
+
+body{
+    font: $base-font-size $base-font;
+    color: $base-color;
+}
+a{
+    color: $base-color;
+}
+.back-link{
+    font-size: 0.8rem;
+    margin-top: 4rem;
+}
+.centered-block__wrapper {
```

Стоит привести к одному виду
```suggestion
@import url("fonts.css");

$base-font:         "Ubuntu", sans-serif;
$base-font-size:    14px;
$base-color:        #1E1E1E;
$accent-color:      #F9F144;
$mark-color:        #E4EDFD;
$ext-color:         #FBFBFB;
$note-color:        #999999;

body {
    font: $base-font-size $base-font;
    color: $base-color;
}
a {
    color: $base-color;
}
.back-link {
    font-size: 0.8rem;
    margin-top: 4rem;
}
.centered-block__wrapper {
```